### ISSUE TITLE
[Bug] Shadows not projected on objects placed at certain heights

### ISSUE THREAD

**Issue by HAZARDU5** [decentraland/feedback#9]
In the parking lot LAND plots at `1,3` `2,3` `1,4` `2,4` I've placed a floating island that's positioned at 30y. I'm assuming this height is above the default light which casts shadows on objects within the LAND. It seems like the direction of projection is from the sun position in the skybox shader.

**Comment by bnolan** [comment]
Yeah, the light is at 64, 64, 64, so things about that won't have shadows. Need a proper shadow solution.
